tests/by-util/test_ls.rs

```diff
@@ -2197,6 +2197,446 @@ fn test_ls_recursive_1() {
         .stdout_is(out);
 }
 
+#[cfg(unix)]
+mod quoting {
```

Comment:
why in "mod quoting" ? :)
thanks

Reply:
Nothing mandatory, I just felt it was more practical to regroup all the tests on special characters in the same place. It might prevent someone to add an unrelated test between those, in the future, thus making the testsuite more organized.

Though, I will remove it if these arguments aren't convincing enough.

Reply:
ok, no worries
could you please add a comment explaining this then ? :) thanks